llvm/unittests/CodeGen/AArch64SelectionDAGTest.cpp

```diff
@@ -796,4 +797,52 @@ TEST_F(AArch64SelectionDAGTest, computeKnownBits_extload_knownnegative) {
   EXPECT_EQ(Known.One, APInt(32, 0xfffffff0));
 }
 
+TEST_F(AArch64SelectionDAGTest,
```

Comment:
Thanks for adding these, they are useful for checking the produced results more precisely. Would it be possible to add something for the BICi changes too?